src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5211,14 +5630,15 @@ public override BoundNode VisitRangeExpression(BoundRangeExpression node)
         public override BoundNode VisitRangeVariable(BoundRangeVariable node)
         {
             var result = base.VisitRangeVariable(node);
+            Visit(node.Value);
```

Comment:
>Visit(node.Value); [](start = 12, length = 18)

Are there potential warnings generated here? If so, please add a test.

Reply:
I change this to disable diagnostics.

---
In reply to: [265725715](https://github.com/dotnet/roslyn/pull/34017#discussion_r265725715) [](ancestors = 265725715)